src/libraries/Microsoft.Extensions.FileProviders.Physical/src/Microsoft.Extensions.FileProviders.Physical.ref.g.cs

```diff
@@ -72,21 +72,20 @@ public partial class PhysicalFilesWatcher : System.IDisposable
         public Microsoft.Extensions.Primitives.IChangeToken CreateFileChangeToken(string filter) { throw null; }
         public void Dispose() { }
         protected virtual void Dispose(bool disposing) { }
-        ~PhysicalFilesWatcher() { }
     }
     public partial class PollingFileChangeToken : Microsoft.Extensions.Primitives.IChangeToken
     {
         public PollingFileChangeToken(System.IO.FileInfo fileInfo) { }
-        public bool ActiveChangeCallbacks { get { throw null; } }
+        public bool ActiveChangeCallbacks { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
         public bool HasChanged { get { throw null; } }
         public System.IDisposable RegisterChangeCallback(System.Action<object?> callback, object? state) { throw null; }
     }
     public partial class PollingWildCardChangeToken : Microsoft.Extensions.Primitives.IChangeToken
     {
         public PollingWildCardChangeToken(string root, string pattern) { }
-        public bool ActiveChangeCallbacks { get { throw null; } }
+        public bool ActiveChangeCallbacks { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
         public bool HasChanged { get { throw null; } }
         protected virtual System.DateTime GetLastWriteUtc(string path) { throw null; }
-        System.IDisposable Microsoft.Extensions.Primitives.IChangeToken.RegisterChangeCallback(System.Action<object?> callback, object? state) { throw null; }
+        System.IDisposable Microsoft.Extensions.Primitives.IChangeToken.RegisterChangeCallback(System.Action<object> callback, object state) { throw null; }
```

Comment:
nullability change cc @eerhardt

Reply:
We are already discussing it here: https://github.com/dotnet/runtime/pull/65775#discussion_r813316598